pkg/interactors/create_user.go

```diff
@@ -0,0 +1,30 @@
+package interactors
+
+import (
+	"github.com/CESARBR/knot-babeltower/pkg/entities"
+	"github.com/CESARBR/knot-babeltower/pkg/logging"
+	"github.com/CESARBR/knot-babeltower/pkg/network"
+)
+
+// CreateUser to interact to user
+type CreateUser struct {
+	logger    logging.Logger
+	userProxy network.UserProxy
+}
+
+// NewCreateUser contructs the interactor
+func NewCreateUser(logger logging.Logger, userProxy network.UserProxy) *CreateUser {
+	return &CreateUser{logger, userProxy}
+}
+
+// Execute runs the use case
+func (cu *CreateUser) Execute(user entities.User) (err error) {
```

Comment:
I was thinking about if we should receive structures as value or as reference. It's better to receive as a reference when the structure is larger but how we define this metric? I'm asking you about it because we need to choose an option and keep the consistency through de code base.

Reply:
There are other things to be analyzed besides changing or not the structure such as the size and further increase. However, as we've discussed it's better to pass value.